### PR TITLE
Add stable_diffusion fp16 and fp32 with args.

### PR DESCRIPTION
**Pull request by pashu123**

### CODE REVIEW

File: shark/examples/shark_inference/stable_diffusion/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+```shell
+pip install diffusers
+pip install scipy
+```
+
+## RUN
+
+```shell
+python --precision="fp32"|"fp16" --prompt="enter the text" --device="cpu"|"cuda"|"vulkan" --import_mlir="True"|"False"
```
Comment (qedawkins) [inline shark/examples/shark_inference/stable_diffusion/README.md:13]: nit: missing `python main.py` here

File: shark/examples/shark_inference/stable_diffusion/main.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        if args.import_mlir == True:
+            return get_vae16(), get_unet16_wrapped()
+        else:
+            return get_shark_model(GCLOUD_BUCKET, VAE_FP16), get_shark_model(
+                GCLOUD_BUCKET, UNET_FP16
+            )
+
+    elif args.precision == "fp32":
+        if args.import_mlir == True:
+            return get_vae16(), get_unet16_wrapped()
```
Comment (qedawkins) [inline shark/examples/shark_inference/stable_diffusion/main.py:34]: This looks like it should be unet32 and vae32 for fp32.

**Review by qedawkins — CHANGES_REQUESTED** [review]
This is a lot cleaner than what we were doing! Can we remove the old script as well? Other than a few comments looks good.

File: shark/examples/shark_inference/stable_diffusion/README.md (hunk 1 above)
Comment (pashu123) [reply shark/examples/shark_inference/stable_diffusion/README.md]: Done.

File: shark/examples/shark_inference/stable_diffusion/main.py (hunk 2 above)
Comment (pashu123) [reply shark/examples/shark_inference/stable_diffusion/main.py]: Done.

**Comment by pashu123** [comment]
@qedawkins I have updated the args. PTAL.

**Review by qedawkins — APPROVED** [review]
LGTM!

**Pull request merged**